docs/unity/part-1.md

```diff
@@ -639,21 +650,26 @@ using SpacetimeDB;
 private float? lastUpdateTime;
 private void FixedUpdate()
 {
-    if ((lastUpdateTime.HasValue && Time.time - lastUpdateTime.Value > 1.0f / movementUpdateSpeed) || !SpacetimeDBClient.instance.IsConnected())
-    {
-        return;
-    }
-
-    lastUpdateTime = Time.time;
-    var p = PlayerMovementController.Local.GetModelPosition();
-    Reducer.UpdatePlayerPosition(new StdbVector3
-        {
-            X = p.x,
-            Y = p.y,
-            Z = p.z,
-        },
-        PlayerMovementController.Local.GetModelRotation(),
-        PlayerMovementController.Local.IsMoving());
+   float? deltaTime = Time.time - lastUpdateTime;
+   bool hasUpdatedRecently = deltaTime.HasValue && deltaTime.Value < 1.0f / movementUpdateSpeed;
```

Comment:
(review note: I asked to switch this to `<`, because as far as I can tell the `>` was a bug)